### PR TITLE
Replacing BSA with Admin Smite

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
1) It's flashier and more exciting

2) Single mob targeted BSA is an immersion ruining lore violation even more than magic lightning bolts are

3) Admins are asking to be able to use smite in the same places BSA are used on the forums, and it'd be redundant, so lets go with the better one.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Will finish this tonight, but didn't want to put in the rest of the work until I know this isn't going to get shut down. There is no code to review so far, only removals.

File: code/game/objects/items/charter.dm [hunk 1]
```diff
@@ -60,7 +60,7 @@
...
 	// Autoapproves after a certain time
 	response_timer_id = addtimer(CALLBACK(src, .proc/rename_station, new_name, user.name, user.real_name, key_name(user)), approval_time, TIMER_STOPPABLE)
-	to_chat(admins, "<span class='adminnotice'><b><font color=orange>CUSTOM STATION RENAME:</font></b>[key_name_admin(user)] (<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A>) proposes to rename the station to [new_name] (will autoapprove in [approval_time / 10] seconds). (<A HREF='?_src_=holder;BlueSpaceArtillery=\ref[user]'>BSA</A>) (<A HREF='?_src_=holder;reject_custom_name=\ref[src]'>REJECT</A>) (<a href='?_src_=holder;CentcommReply=\ref[user]'>RPLY</a>)</span>")
+	to_chat(admins, "<span class='adminnotice'><b><font color=orange>CUSTOM STATION RENAME:</font></b>[key_name_admin(user)] (<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A>) proposes to rename the station to [new_name] (will autoapprove in [approval_time / 10] seconds). (<A HREF='?_src_=holder;adminsmite=\ref[user]'>SMITE</A>) (<A HREF='?_src_=holder;reject_custom_name=\ref[src]'>REJECT</A>) (<a href='?_src_=holder;CentcommReply=\ref[user]'>RPLY</a>)</span>")
```
Comment (lzimann) [inline code/game/objects/items/charter.dm]: Use the define for smite: `ADMIN_SMITE(user)` and for the `ADMIN_QUE(user)` for the `?`

**Comment by Shadowlight213** [comment]
:-1: I just looked over smite, and it just deals 75 burn damage.
BSA is much better as it only deals enough damage to take them 1 point above crit on the first shot.
I don't want to have to worry about healing someone out of crit every time.
Plus, smite doesn't make much sense for centcom stuff. You're supposed to be talking to central command, not the gods.

**Comment by KorPhaeron** [comment]
Centcomm being able to magically explode (without damaging anything else around the target) any living being in the universe doesn't make very much sense given what we know and do in the rest of the setting either.

BSA is supposed to be a giant fucking cannon not just turning people off. If we're going to fuck with the immersion we may as well go all the way and say "yeah some magical thing hit you with a lightning bolt"

**Comment by ExcessiveUseOfCobblestone** [comment]
just BS it with they took the Tesla gun from RND and tweaked it to be a long range, high accuracy weapon or something.

**Comment by KorPhaeron** [comment]
why would we have a security force or ert or deathsquad or even a game if they can kill anyone like that though

**Comment by jughu** [comment]
Or you can making smiting just the BSA for prayers and BSA for centcom messages/etc so no lpre gets violated

**Comment by Dorsisdwarf** [comment]
Make the smite for centcomm message hyperlinks just be the comms console firing a huge bolt of lightning at you.

**Comment by Shadowlight213** [comment]
Comms console lightning could be interesting.

**Comment by Jordie0608** [comment]
It's a gameplay abstraction, lore shouldn't factor into this at all.

**Comment by Cyberboss** [comment]
>Fire the BSA, single reactor ignition only

**Comment by Thunder12345** [comment]
Make BSA an option for smiting

**Comment by PKPenguin321** [comment]
Do what thunder said

**Comment by KorPhaeron** [comment]
Stale

**Pull request closed without merging**